controller/build/build.go

```diff
@@ -133,6 +133,15 @@ func RunBuild(ctx context.Context, dockerCli command.Cli, in controllerapi.Build
 		}
 	}
 
+	// ENV-guarded ability load images by default if no other output is specified.
+	// This is useful when migrating from Docker buildx drivers to others, as it aligns the default behavior.
+	if build.DefaultLoad() && len(outputs) == 0 && len(opts.Tags) > 0 {
```

Comment:
why the `opts.Tags` check in here?

Reply:
My thinking is:
Loading an image from a remote driver can be slow as networking is involved.
So, I'd like to differentiate the case where I just want to check if a build passes (and populate the cache of the remote build driver) vs I tag the image so that I can reuse it later.